LoRaEngine/modules/LoRaSimulator/LoRaSimulator/LoRaSimulator.csproj

```diff
@@ -9,6 +9,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="testconfig - Copy.json">
```

Comment:
Is the copy of the Copy file normal?

Reply:
hum, that should not be present. I guess commit went before the project file has been saved in VS. Please remove this part. thank you